src/components/AlertModal/AlertModal.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { FunctionComponent, useMemo } from "react";
+import { Modal, StyleSheet, Text, View } from "react-native";
+import AlertLogo from "../../../assets/icons/alert.svg";
+import { DarkButton } from "../Layout/Buttons/DarkButton";
+import { SecondaryButton } from "../Layout/Buttons/SecondaryButton";
+import { DangerButton } from "../Layout/Buttons/DangerButton";
+import { size } from "../../common/styles";
+
+const styles = StyleSheet.create({
+  centeredView: {
+    flex: 1,
+    width: "100%",
+    justifyContent: "center",
+    alignItems: "center",
+    alignSelf: "center",
```

Comment:
```suggestion
```

Reply:
good catch. Not sure why this was added in the first place. Updated